en/bitcoin-core/index.md

```diff
@@ -113,4 +113,8 @@ if ( $( window ).width() > 400 && $( window ).height() > 600 ) {
 }
 </script>
 
+#### Disclaimer
+
+"Bitcoin Core" is a separate project from bitcoin.org and the views expressed by this website and its owners, are not necessarily the views expressed by the Bitcoin Core project or its developers.
```

Comment:
Move the comma back to before the sentence conjunction: "Bitcoin Core" is a separate project from bitcoin.org*,* and the views expressed by this website and its owners are not necessarily the views expressed by the Bitcoin Core project or its developers.